app/uk/gov/hmrc/gform/LookupLoader.scala

```diff
@@ -102,19 +108,42 @@ class LookupLoader {
     keywords: String,
     priority: String,
     region: String,
+    inEU: String,
+    inEEA: String,
+    inEFTA: String,
```

Comment:
I've thought this would be `Map[String, String]` and it will contain all columns from cvs. Let'a hope we won't be adding these new columns every week 🤞